src/gt4py/backend/debug_backend.py

```diff
@@ -160,7 +160,7 @@ def visit_If(self, node: gt_ir.If):
         return ["".join([str(item) for item in line]) for line in body_sources.lines]
 
 
-class DebugModuleGenerator(gt_backend.BaseModuleGenerator):
+class DebugModuleGenerator(gt_backend.DebugExtModuleGenerator):
```

Comment:
As I commented in the definition of `DebugExtModuleGenerator` I don't see the point of adding that class.

```suggestion
class DebugModuleGenerator(gt_backend.BaseModuleGenerator):
```